Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -29,15 +28,12 @@
 #import <BraintreeCore/BTAPIClient_Internal.h>
 #import <BraintreeCore/BTPaymentMethodNonceParser.h>
 #import <BraintreeCore/BTLogger_Internal.h>
-#import <BraintreePayPal/BTPayPalRequest.h>
 #import <BraintreePayPal/BTConfiguration+PayPal.h>
 #import <BraintreePayPal/BTPayPalLineItem.h>
 #import <PayPalDataCollector/PayPalDataCollector-Swift.h>
 #endif
 
 NSString *const BTPayPalDriverErrorDomain = @"com.braintreepayments.BTPayPalDriverErrorDomain";
-NSString *const BTCallbackURLHostAndPath = @"onetouch/v1/";
```

Comment:
do we still need one touch? My memory is probably horribly off but I thought that's a 💀  integration

Reply:
[One Touch](https://www.paypal.com/us/webapps/mpp/one-touch-checkout) is still a thing. You're probably thinking of the code from the OneTouch SDK that was added into our SDK a long time ago and which we recently deleted.